go/base/management/commands/go_manage_routing_table.py

```diff
@@ -51,48 +43,46 @@ def handle(self, *args, **options):
                 else:
                     raise CommandError('Please provide %s:' % (opt.dest,))
 
-        command_options = [c for c in self.CONFLICTING_OPTIONS if options[c]]
-        if len(command_options) != 1:
+        # Ensure a single option is chosen from the local options
+        cmd_options = [c.dest for c in self.LOCAL_OPTIONS if options[c.dest]]
+        if len(cmd_options) != 1:
             raise CommandError('Please provide exactly one of: %s' % (
-                ['--%s' % c for c in self.CONFLICTING_OPTIONS],))
-
-        user = get_user_by_email(options['email-address'])
-        user_api = vumi_api_for_user(user)
+                ['--%s' % c.dest for c in self.LOCAL_OPTIONS],))
```

Comment:
It would be nice if these were actual commands (in the `BaseGoCommand` sense). But they don't quite fit that pattern. I guess `CONFLICTING_OPTIONS` is less clear but it just seems a bit confusing when *all* the options "conflict". Maybe we could call it something like `COMMANDS` and then leave a comment explaining there must only be 1 command?